newspaper/extractors.py

```diff
@@ -148,8 +148,10 @@ def parse_byline(search_str):
 
         for match in matches:
             content = ''
-            if match.tag == 'meta':
+            if match.tag == 'meta' or match.tag == 'span' or match.tag == 'a':
                 mm = match.xpath('@content')
+                if not mm:
+                    mm=str(match.text_content()).split()
```

Comment:
Confident that this will increase the recall but will precision take a hit?

I am going to approve this but can you also add some urls where you found that these changes were helpful